app/lib/settings_helper.rb

```diff
@@ -29,6 +29,10 @@ def self.relays_enabled_for_reservation?
     setting "relays.#{Rails.env}.reservation_enabled"
   end
 
+  def self.active_storage_enabled?
+    feature_on?(:active_storage) || feature_on?(:active_storage_for_images_only)
```

Comment:
I think I'm leaning towards letting each feature flag do its own thing - what's your thinking about intertwining here?  

If we keep this logic I'd vote for renaming to `active_storage_for_images_enabled?`

Reply:
Because of how the flags are named, it seemed like `active_storage` could/would be a superset of `active_storage_for_images_only`. Also doing it this way would mean no change in OSU's settings.yml. Neither of these may be interesting enough reasons, don't have a strong opinion.